CMSgov/easi-app
# [EASI-3232] Select resolution page

# EASI-3232

[Figma
](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=1%3A24513&mode=design&t=4vYeXeppt9vfSOSj-1)

## Changes and Description

- Created Select Resolution view (src/views/GovernanceReviewTeam/Actions/Resolutions.tsx)
  - Conditional logic for rendering title, description, and resolution options (see [Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=1%3A25685&mode=design&t=qtMtNyeCNLabcHmc-1))
  - Summary box renders only available resolutions
- Routes for each resolution
  - Issue a Life Cycle ID (/governance-review-team/:systemIntakeId/resolutions/issue-lcid)
  - Not an IT Governance request (/governance-review-team/:systemIntakeId/resolutions/not-it-request)
  - Not approved by GRB  (/governance-review-team/:systemIntakeId/resolutions/not-approved)
  - Close request  (/governance-review-team/:systemIntakeId/resolutions/close-request)
  - Re-open request  (/governance-review-team/:systemIntakeId/resolutions/re-open-request)
- Basic page layout for resolution forms
  - Added `<ActionForm>` component
  - Created "Selected resolution" box with action title and "Change resolution" link



## How to test this change

- Make sure `itGovV2Enabled` flag is enabled
- Go to /governance-review-team/:systemIntakeId/actions
- Select "Issue a decision or close this request"
- Ensure Resolutions page works as expected



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
